### PR TITLE
Add FR-NL-1.05 validation to NL validation plugin

### PR DESCRIPTION
**Pull request by derekgengenbacher-wf**
#### Reason for change
Added FR-NL-1.05 validation to NL validation plugin

#### Description of change
Validation checks that the character encoding UTF-8 MUST be used in the filing instance document

#### Steps to Test
CI

**review**:
@Arelle/arelle

### CODE REVIEW

**Comment by aviary-wf** [comment]
## Security Insights
No security relevant content was detected by automated scans.

## Action Items
* Review PR for [security impact](https://wiki.atl.workiva.net/display/SECURITY/Development+Security+Review+Guidelines); comment "security review required" if needed or unsure
* Verify [`aviary.yaml`](https://wiki.atl.workiva.net/pages/viewpage.action?pageId=42762752#SecurityInsightFramework&Tooling-ConfiguringAviary) coverage of security relevant code
---
_Questions or Comments? Reach out on Slack: #support-infosec._

File: tests/resources/conformance_suites/nl_nt18/fr_nl/1-05-testcases.xml [hunk 1]
```diff
@@ -0,0 +1,22 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<?xml-stylesheet type="text/xsl" href="../testcase.xsl"?>
+<testcase
+        xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+        xmlns="http://xbrl.org/2005/conformance"
+        name="NL.FR-NL-1.06"
```
Comment (aaroncameron-wk) [inline tests/resources/conformance_suites/nl_nt18/fr_nl/1-05-testcases.xml:6]:
```suggestion
        name="NL.FR-NL-1.05"
```

File: tests/resources/conformance_suites/nl_nt16/fr_nl/1-05-testcase.xml [hunk 2]
```diff
@@ -0,0 +1,22 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<?xml-stylesheet type="text/xsl" href="../testcase.xsl"?>
+<testcase
+        xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+        xmlns="http://xbrl.org/2005/conformance"
+        name="NL.FR-NL-1.06"
```
Comment (aaroncameron-wk) [inline tests/resources/conformance_suites/nl_nt16/fr_nl/1-05-testcase.xml:6]:
```suggestion
        name="NL.FR-NL-1.05"
```

**Review by aaroncameron-wk — CHANGES_REQUESTED** [review]

File: tests/resources/conformance_suites/nl_nt18/fr_nl/1-05-testcases.xml (hunk 1 above)
Comment (derekgengenbacher-wf) [reply tests/resources/conformance_suites/nl_nt18/fr_nl/1-05-testcases.xml:6]: Garr... good catch!

**Review by allenbross-wf — APPROVED** [review]

**Pull request merged**